docs/PfSense.md

```diff
@@ -0,0 +1,20 @@
+In order to boot PfSense via PXE and use unattended install,
+you need the following:
+
+1. Extracted PfSense installation ISO exported via NFS.
```

Comment:
@pkubaj please provide link to PfSense image you tested, so there will be no confusion in future. Please provide method of extracting or link to relevant documentation in the web.